components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2782,7 +2807,7 @@ where
         }
         if !state.pending_entries.is_empty() {
             self.delegate
-                .handle_raft_committed_entries(ctx, state.pending_entries);
+                .handle_raft_committed_entries(ctx, state.pending_entries.drain(..));
```

Comment:
Why change it?

Reply:
Because I want to impl drop trait for Apply, so I change the handle_raft_committed_entries fn.